server/src/main/java/org/opensearch/index/translog/Translog.java

```diff
@@ -614,7 +615,11 @@ final Checkpoint getLastSyncedCheckpoint() {
 
     // for testing
     public Snapshot newSnapshot() throws IOException {
-        return newSnapshot(0, Long.MAX_VALUE);
+        return newSnapshot(0, Long.MAX_VALUE, false);
```

Comment:
Use the helper method added.

```suggestion
        return this.newSnapshot(0, Long.MAX_VALUE);
```